toolkit/themes/shared/design-system/README.design-tokens-json.stories.md

```diff
@@ -0,0 +1,59 @@
+# Design tokens
+## Source of truth
+### Background
+The benefit of storing design tokens with a platform-agnostic format such as JSON is that it can be converted or translated into other languages or tools (e.g CSS, Swift, Kotlin, Figma).
```

Comment:
```suggestion
### Background
`design-tokens.json` is the source of truth for all our design tokens. The benefit of storing design tokens with a platform-agnostic format such as JSON is that it can be converted or translated into other languages or tools (e.g CSS, Swift, Kotlin, Figma).
```